packages/web-react/src/components/Alert/README.md

```diff
@@ -10,13 +10,17 @@ import { Alert } from '@lmc-eu/spirit-web-react/components';
 <Alert color="success">Hey! Pay attention!</Alert>
 <Alert color="danger">Hey! Pay attention!</Alert>
 <Alert color="informative">Hey! Pay attention!</Alert>
+<Alert color="informative" iconName="warning">Hey! Pay attention!</Alert>
+<Alert color="informative" isCentered>Hey! Pay attention!</Alert>
 ```
 
 ## Available props
 
-| Name       | Type                               | Description                               |
-| ---------- | ---------------------------------- | ----------------------------------------- |
-| `children` | `any`                              | Content of the Alert                      |
-| `color`    | `success`, `danger`, `informative` | Color of the component, default `success` |
+| Name         | Type                               | Description                               |
+| ------------ | ---------------------------------- | ----------------------------------------- |
+| `children`   | `any`                              | Content of the Alert                      |
+| `color`      | `success`, `danger`, `informative` | Color of the component, default `success` |
+| `iconName`   | `string`                           | Icon used in Alert                        |
```

Comment:
Thinking about whether simply `icon` prop will be better or not 🤔

Reply:
I think not. Icon is the component iconName is the name of the Icon used. Here we define the icon name used in Alert, not the component.

Reply:
🆗